### DISCUSSION TITLE
Where is the data stored/how could I create backups?

### DISCUSSION THREAD

**Discussion by JakobStr** [hcengineering/platform#6403] [Architecture]
I want to host my own instance of this project with https://github.com/hcengineering/huly-selfhost.
How could I create backups, in case my PC crashes? 
And could they easily be moved to a new PC? 
My problem is, that I cannot use any software like JIRA etc., because we have very strict data security laws here in germany.
Therefore I am also not allowed to use huly.app, because it could transfer data from my company out of the country.
Really appreciate your work!

**Comment by lexiv0re** [comment]
@JakobStr please check a related q/a https://github.com/hcengineering/platform/discussions/6389 for the second part of your question.

**Reply by JakobStr** [reply]
Thanks for your response @lexiv0re.
I read this post before asking my question. 
But I still don't understand it. 
Is the data stored in a database like mongodb/sql etc.? 
And how could I backup this data? Is there maybe a functionality for this already?
Thanks in advance.

**Comment by rpolitex** [comment]
@JakobStr regarding backup

I've found an answer in Huly Slack:
"This <DataBase> dump works only from Desktop app now, please use desktop app if you want to try: https://huly.io/download"
![image](https://github.com/user-attachments/assets/dddb7615-ddc5-406f-8dc8-86b354066769)
